Source/WebCore/editing/cocoa/AttributedString.h

```diff
@@ -40,10 +41,10 @@
 #define PlatformNSPresentationIntent    NSPresentationIntent.class
 #define PlatformNSShadow                NSShadow.class
 #define PlatformNSTextAttachment        NSTextAttachment.class
-#define PlatformNSTextList              NSTextList
+#define PlatformNSTextList              NSTextList.class
```

Comment:
Just curious, why is this .class needed now?

Reply:
Oh, good catch! We only need this change for the `PlatformNSTextTableBlock` define, not text lists.